### ISSUE TITLE
batch_size Can I set it to a different value?

### ISSUE THREAD

**Issue by huwenfeng233** [Finspire13/DiffAct#4]
Hi  Daochang 
thank you for sharing the code. I noticed an issue in line 69 of the main.py file, where batch_size=1 is hard-coded, and it doesn't utilize the batch_size value specified in the command-line arguments. I understand that when I try to manually modify the batch_size value, the code does not run properly. Typically, the batch_size value is not limited to 1. I am wondering if there is a version of the code that supports a batch_size value other than 1.

**Comment by Finspire13** [comment]
Hi,

You can change the batch_size in the config file to use a different actual batch size for training. The current implementation of batch training is based on gradient accumulation (https://github.com/Finspire13/DiffAct/blob/main/main.py#L121) to handle different lengths of videos.

**Comment by huwenfeng233** [comment]
Thanks！
